src/modules/shared/pages/innovation/support/wizard-support-summary-progress-update-milestones/support-summary-progress-update-milestones.component.ts

```diff
@@ -0,0 +1,469 @@
+        }
+      })
+    );
+
+    this.wizard.addStep(
+      new WizardStepModel<DescriptionStepInputType, DescriptionStepOutputType>({
+        id: 'descriptionStep',
+        title: `Add a description to your progress update`,
+        component: WizardInnovationSupportSummaryProgressUpdateMilestonesDescriptionStepComponent,
+        data: {
+          selectedCategories: [],
+          otherCategory: null,
+          selectedSubcategories: [],
+          description: '',
+          file: null,
+          fileName: ''
+        },
+        outputs: {
+          previousStepEvent: data =>
+            this.onPreviousStep(data, this.onDescriptionStepOut, this.onSubcategoriesStepIn, this.onCategoriesStepIn),
+          nextStepEvent: data => this.onNextStep(data, this.onDescriptionStepOut, this.onDateStepIn)
+        }
+      })
+    );
+
+    this.wizard.addStep(
+      new WizardStepModel<DateStepInputType, DateStepOutputType>({
+        id: 'dateStep',
+        title: `Select a date for this progress update`,
+        component: WizardInnovationSupportSummaryProgressUpdateMilestonesDateStepComponent,
+        data: {
+          day: '',
+          month: '',
+          year: ''
+        },
+        outputs: {
+          previousStepEvent: data => this.onPreviousStep(data, this.onDateStepOut, this.onDescriptionStepIn),
+          nextStepEvent: data => this.onNextStep(data, this.onDateStepOut, this.onSummaryStepIn)
+        }
+      })
+    );
+
+    this.wizard.addStep(
+      new WizardStepModel<SummaryStepInputType, null>({
+        id: 'summaryStep',
+        title: `Check your progress update`,
+        component: WizardInnovationSupportSummaryProgressUpdateMilestonesSummaryStepComponent,
+        data: {
+          categoriesStep: {
+            categories: [],
+            otherCategory: null
+          },
+          subcategoriesStep: {
+            subcategories: []
+          },
+          descriptionStep: {
+            description: '',
+            file: null,
+            fileName: ''
+          },
+          date: ''
+        },
+        outputs: {
+          previousStepEvent: data => this.onPreviousStep(data, this.onDateStepIn),
+          submitEvent: data => this.onSubmit(data),
+          goToStepEvent: stepId => this.onGoToStep(stepId)
+        }
+      })
+    );
+
+    this.setPageStatus('READY');
+  }
+
+  getOrgCategories(): {
+    name: string;
+    description: string;
+  }[] {
+    return SUPPORT_SUMMARY_MILESTONES[this.userOrgAcronym].map(org => {
+      return {
+        name: org.name,
+        description: org.description
+      };
+    });
+  }
+
+  getCategorySubcategories(): {
+    name: string;
+    description: string;
+  }[] {
+    return (
+      SUPPORT_SUMMARY_MILESTONES[this.userOrgAcronym].filter(
+        org => org.name === this.wizard.data.categoriesStep.categories[0]?.name
+      )[0]?.subcategories ?? []
```

Comment:
If I understood correctly this could be a find instead of using filter and having to deal with `[0]`.